### PR TITLE
Add ingress host

### PR DESCRIPTION
**Pull request by KiranVarma02**
### Change description ###
Add ingress host - commit from expressjs-template

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[* ] No
```

### CODE REVIEW

**Review by timja — DISMISSED** [review]

File: charts/adoption-frontend/values.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 nodejs:
   image: hmctspublic.azurecr.io/ado/adoption-frontend:latest
+  ingressHost: adoption-frontend.{{ .Values.global.environment }}.platform.hmcts.net
```
Comment (timja) [inline charts/adoption-frontend/values.yaml]: you need to bump the chart version as well, and looks like you should remove frontend from the url based on https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/adoption/adoption-frontend.yaml#L28

```suggestion
  ingressHost: adoption.{{ .Values.global.environment }}.platform.hmcts.net
```

Comment (KiranVarma02) [reply charts/adoption-frontend/values.yaml]: i have already updated chart version in different PR. I think i am going to leave it there as its going to be merged very soon after this.

Comment (KiranVarma02) [reply charts/adoption-frontend/values.yaml]: looks like i have to update that now for pipeline to work.  I will do that now.

Comment (KiranVarma02) [reply charts/adoption-frontend/values.yaml]: done

**Review by Jamesnelson117 — APPROVED** [review]
LGTM but maybe re-check with @timja

**Pull request merged**